micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -0,0 +1,251 @@
+
+## [1.8.16]  [23-Jan-2025]
+- FIxed some of FEATURE/HCMPRE-1852 :: accessibility issues of image(#2120) 
+
+## [1.8.15]  [21-Jan-2025]
+- Removed support for any new context path to have employee linked in the url.To use this,the new context path should be linked with employee. 
+
+## [1.8.13]  [16-Dec-2024]
+- Made validations for name, mobile number, and password fields in user profile update screen configurable through MDMS data
+  - Implemented dynamic regex validation for profile updates based on MDMS data
+  - Added support for custom regex patterns through `UserProfileValidationConfig`
+  - Example MDMS data: https://github.com/egovernments/egov-mdms-data/blob/UNIFIED-QA/data/mz/commonUIConfig/UserProfileValidationConfig.json
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)